services/spar/test-integration/Util/Core.hs

```diff
@@ -287,6 +289,24 @@ aFewTimes action good = do
       (\_ -> pure . not . good)
       (\_ -> action `runReaderT` env)
 
+-- | Duplicate of 'Spar.Intra.Brig.getBrigUser'.
+getUserBrig :: HasCallStack => UserId -> TestSpar (Maybe User)
+getUserBrig uid = do
+  env <- ask
+  let req =
+        (env ^. teBrig) . path "/self"
```

Comment:
Prefer if we stop calling "user-facing" endpoints from service to service. We should instead only use internal endpoints between services

Reply:
ah, i agreed with you when you wrote this, but i just realized that luckily this is not service-to-server, but integration-test-to-service.  i suppose that's legit?

Reply:
ah.  right.  so, i'm happy to take on the task of refactoring this properly, over our entire code base.  but that's a separate concern, and the helper in the tests can stay!  :)